### ISSUE TITLE
Silex menu not visible 

### ISSUE THREAD

**Issue by TeaLu** [silexlabs/Silex#504]
I'm using the editor from the main silex website, not self-hosted and the top menu does not appear and I cannot load any templates or files. Is there a solution for this?

**Comment by lexoyo** [comment]
hello
maybe this is normal, we changed the UI recently and it should look like this:
![screenshot from 2016-09-27 15-55-37](https://cloud.githubusercontent.com/assets/715377/19568857/9b97c2f6-96f3-11e6-82fd-6ae7122f1c9f.png)

**Comment by TeaLu** [comment]
yes that is how it looks, and if i want to upload a template what should i access?

**Comment by lexoyo** [comment]
If you have downloaded the template, you can put its file on Owncloud, Dropbox, FTP, or Github and then open it from silex

**Comment by oopiicaa** [comment]
how can I access the Apollo mode?

**Comment by lexoyo** [comment]
hello
Silex has evolved and the apollo mode is no more... It is always Apollo mode now
Can I help you with something?

**Comment by oopiicaa** [comment]
sorry then, was confused by some outdated guides. thank you

**Comment by lexoyo** [comment]
:+1:

**Comment by ckmsmission** [comment]
I'm using the editor from the main silex website, not self-hosted and the top menu does not appear and I cannot load any templates or files. Is there a solution for this?please give the proper guidance for this question

**Comment by ckmsmission** [comment]
when iam putting my downloaded file it is showing the message"CLICK HERE to authorize Cloud Explorer to use your Dropbox account.".How to solve this
